tests/integration/conftest.py

```diff
@@ -87,7 +87,12 @@ async def microk8s_kubernetes_cloud_and_model(ops_test: OpsTest, microk8s_applic
     model_name = f"k8s-{ops_test._generate_model_name()}"
 
     try:
-        LOG.info("Add cloud %s on controller %s", JUJU_CLOUD_NAME, ops_test.controller_name)
+        LOG.info(
+            "Add cloud %s on controller %s and model name %s",
+            JUJU_CLOUD_NAME,
+            ops_test.controller_name,
+            model_name,
+        )
```

Comment:
is this needed?

Reply:
Just some extra info on the model. I could not find it, and this made my debugging a bit easier.